packages/async-apollo-provider/test/__tests__/AsyncApolloProvider.test.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { ApolloClient } from 'apollo-client';
+import { ApolloProvider } from 'react-apollo';
```

Comment:
@ej9x Why we should import it here?

Reply:
Because we should check that the ApolloProvider will be added to the jsx after fetching the schema